students/AndrewMiotke/session03/list_lab.py

```diff
@@ -1,21 +1,49 @@
 #!/usr/bin/env python3
 # https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/list_lab.html
 
-def series_one():
-    fruits = ["Apples", "Pears,", "Oranges", "Peaches"]
-    print(fruits)
+# Series One
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruit_list)
 
-    add_fruit = input("Please add another fruit here: ")
-    fruits.append(add_fruit)
-    print(fruits)
+add_fruit = input("Type another fruit: ")
+fruit_list.append(add_fruit)
+print(fruit_list)
 
-    add_number = input("Please add a number: ")
-    for number in fruits:
-        fruits[0].append(add_number)
+get_index = int(input("Type a number to get a fruit: "))
+print(fruit_list[get_index - 1])
 
-    print(fruits)
+new_fruit_list = [add_fruit] + fruit_list
+print(new_fruit_list)
 
+insert_fruit = input("Insert a fruit: ")
+fruit_list.insert(0, insert_fruit)
+print(fruit_list)
 
-if __name__ == "__main__":
+for fruits_begining_with_p in fruit_list:
+    if fruits_begining_with_p[0].lower() == "p":
+        print(fruits_begining_with_p)
 
-    series_one()
\ No newline at end of file
+# Series Two
+print(fruit_list)
+
+fruit_list.pop()
+print(fruit_list)
+
+delete_fruit = input("What fruit would you like to delete? ")
+fruit_list.remove(delete_fruit)
+print(fruit_list)
+
+# Series Three
+while True:
+    for favorite_fruit in fruit_list:
```

Comment:
Be careful here, if you remove the fruit while iterating, you'll actually skip head. Try running this and see the result when you remove a fruit (you'll end up skipping the next one in line)

Reply:
To resolve this, you'll want to iterate through the list in a way which lets you keep track of the index